boards/v2/qemu/qemu_x86/qemu_x86_nokpti.yaml

```diff
@@ -1,4 +1,4 @@
-identifier: qemu_x86_nokpti
+identifier: qemu_x86/ia32/nokpti
```

Comment:
as above

Reply:
> as above

@nordicjm,  isn't `qemu_x86/ia32/nokpti` a full identifier already ?

Reply:
yes, marked as resolved for this